src/CMakeLists.txt

```diff
@@ -1,19 +1,41 @@
 
 cmake_minimum_required(VERSION 3.1)
 
-project(Iaito VERSION 1.0.0)
+project(iaito VERSION 1.0.0)
 
-set(CMAKE_CXX_STANDARD 11)
+set(QT_MIN_VERSION "5.6.0")
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+include(CheckCXXCompilerFlag)
+include(FeatureSummary)
+include(GenerateExportHeader)
+
+find_package(Qt5 ${QT_MIN_VERSION} NO_MODULE REQUIRED
+	WebEngine
+	WebEngineWidgets
+)
+
+find_package(Radare2 1.4.0 REQUIRED)
+include_directories(${RADARE2_INCLUDE_DIRS})
+
+feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_INSTALL_PREFIX ${BUILD_DIR})
```

Comment:
Ah, I see! Then I guess that's because `${CMAKE_INSTALL_PREFIX}/bin` will become `/bin`.
Have you tried the following?
```
set(BIN_INSTALL_DIR bin)
install(TARGETS iaito RUNTIME DESTINATION ${BIN_INSTALL_DIR})
```
So install will only receive "bin" without the slash in front of it and use `CMAKE_INSTALL_PREFIX` automatically.